### ISSUE TITLE
featuredImage does not show for non-index.md pages

### ISSUE THREAD

**Issue by mikijov** [josephhutch/aether#39]
I have articles in many places. For example
/content/page1.md
/content/post/page2.md
/content/post/page3/page3.md
/content/post/page4/index.md
Even when they have identical content, only page4 featuredImage will show up. I am getting no relevant warnings or other messages when running hugo with --verbose --debug.

**Comment by josephhutch** [comment]
Since aether uses the Hugo concept of [Page Bundles](gohugo.io), only articles named index.md in a leaf will include the featured image.  [Page Resources](https://gohugo.io/content-management/page-resources/) can only be accessed from within these Leaf Bundles so I cannot make any changes to support this content hierarchy.  Let me know if you run into any trouble correcting this issue in your site or if you have any other questions.

**Comment by mikijov** [comment]
Hmm... Even though I submitted couple of PR's my knowledge of Hugo is only superficial, so I am not confident making design suggestions... The old way of using featuredImage worked. I am curious, is there a way to have separate layout/template for non-resource-bundle pages? Maybe continue using featuredImage there, or find some other way to add images to them.

The main reason I ask is that using the old way, I was able to control URL of some of the better known pages, like
www.xxx.com/license
www.xxx.com/download
and similar. Now they lose images. I tried renaming content/license.md to content/license/index.md, but that causes a warning in Hugo:
```
WARN 2019/08/09 05:24:44 found no layout file for "HTML" for "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
```
and I get empty home page (no heading or anything). Can you make any recommendation how to get any posts specific url, like www.xxx.com/license? I do not mind digging some if you can point me in the right direction.

**Comment by josephhutch** [comment]
I will change the post template to work for all single pages.  That way, renaming content/license.md to content/license/index.md will solve the issue.  Thank you for bringing this to my attention.

**Comment by mikijov** [comment]
Hi Joseph. No (big) rush on this, but I am curious if you could make a guess at when you might fix this? I am trying to decide if I should wait few days or if I should go to the older version of Aether. I cannot update my site with latest code without breaking something.

**Comment by josephhutch** [comment]
Sorry for the delay @mikijov.  I misunderstood your issue so the fix I had in mind would not work.  I think the issue may be in your site layout.  Do you have your site source somewhere I could take a look?  Renaming 'content/license.md' to 'content/license/index.md' should solve your issue and you should not get that warning.

**Comment by mikijov** [comment]
It's a private repo at https://bitbucket.org/mikijov/haker-site. If you have an account there I can give you access.

**Comment by mikijov** [comment]
OK, I reproduced the problem on a new site. I simply created new Hugo site `hugo new site aether-test`, added theme `git clone https://github.com/josephhutch/aether.git themes/aether` and added single content page in `content/intro/index.md` with an image `content/intro/wopr.jpg`. The result site is attached.
[aether-test.tar.gz](https://github.com/josephhutch/aether/files/3500324/aether-test.tar.gz)

When launching, I get this:
```
Building sites … WARN 2019/08/14 03:46:18 found no layout file for "HTML" for "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2019/08/14 03:46:18 found no layout file for "HTML" for "taxonomyTerm": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2019/08/14 03:46:18 found no layout file for "HTML" for "taxonomyTerm": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.

                   | EN  
+------------------+----+
  Pages            |  3  
  Paginator pages  |  0  
  Non-page files   |  0  
  Static files     |  0  
  Processed images |  0  
  Aliases          |  0  
  Sitemaps         |  1  
  Cleaned          |  0  

Total in 20 ms
Watching for config changes in 
Environment: "development"
Serving pages from memory
Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
Web Server is available at //localhost:1313/ (bind address 127.0.0.1)
Press Ctrl+C to stop
```
and the `localhost:1313` is empty.

**Comment by mikijov** [comment]
Thanks @josephhutch. Your latest commit fixes my problem, so top level content pages work `/content/page-name/index.md`.
